src/test/java/org/mutabilitydetector/unittesting/matchers/reasons/WithAllowedReasonsMatcherTest.java

```diff
@@ -157,8 +157,6 @@ public void mismatchDescriptionExplicitlyStatesNoReasonsHaveBeenAllowed() throws
         MutableReasonDetail disallowedReason = newMutableReasonDetail("disallowed", unusedCodeLocation, unusedReason());
         
         Matcher<MutableReasonDetail> onlyAllowOneReason = mock(Matcher.class);
-        when(onlyAllowOneReason.matches(disallowedReason)).thenReturn(false);
```

Comment:
So the `onlyAllowOneReason` declaration on line 159 should be dropped as well, right?

Reply:
Since this mock is only invoked in the stubbing and the stubbing is already removed, the declaration should be removed as well.

Reply:
Since this mock is only invoked in the stubbing and the stubbing is already removed, the declaration should be removed as well.